bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -2337,7 +2364,7 @@ public function previewAction(Request $request)
                 }
             }
         } elseif ($linkGenerator = $object->getClass()->getLinkGenerator()) {
-            $url = $linkGenerator->generate($object, ['preview' => true, 'context' => $this]);
+            $url = $linkGenerator->generate($object, [['preview' => true, 'context' => $this]]);
```

Comment:
on purpose?

Reply:
yes, well, I started implementing that from another project and just copied it into my Pimcore Core fork :D, I will fix that later.

Reply:
fixed